packages/shared/src/components/Settings.tsx

```diff
@@ -116,11 +116,11 @@ export default function Settings({
     <div className={classNames('flex', 'flex-col p-6', className)} {...props}>
       <Section className="mt-0">
         <SectionTitle>Layout</SectionTitle>
-        <IconsSwitch
+        <CustomSwitch
```

Comment:
Just thinking maybe "TextToggle" is a better name for it?
Custom doesn't really describe what it is.

Reply:
This can also work on any kind of component that accepts `className` and texts. Can be an Icon any random component. WDYT of `GenericSwitch`?

Reply:
Maybe just Switch or Toggle even? 👀

Reply:
We already have the `Switch` - the most basic one we have 😭

Reply:
Alright let's keep CustomSwitch then :)